docs/how-to/contribute-code.rst

```diff
@@ -84,27 +84,27 @@ Next, install any additional dependencies for your operating system:
 
     .. code-block:: console
 
-      (venv) $ sudo apt-get update
-      (venv) $ sudo apt-get install python3-dev libgirepository1.0-dev libcairo2-dev libpango1.0-dev libwebkit2gtk-4.0-37 gir1.2-webkit2-4.0
+      (venv) $ sudo apt update
+      (venv) $ sudo apt install pkg-config python3-dev libgirepository1.0-dev libcairo2-dev gir1.2-webkit2-4.0 libcanberra-gtk3-module
 
     Fedora
 
     .. code-block:: console
 
-      (venv) $ sudo dnf install pkg-config python3-devel gobject-introspection-devel cairo-devel cairo-gobject-devel pango-devel webkitgtk3
+      (venv) $ sudo dnf install pkg-config python3-devel gobject-introspection-devel cairo-gobject-devel webkitgtk4 libcanberra-gtk3
```

Comment:
Is `webkitgtk4` correct, or a typo? I just noticed it here, but I can see it's on the main beeware tutorial as well.

Reply:
It is technically correct...but there is apparently a better option.

The `webkitgtk4` package was [renamed](https://src.fedoraproject.org/rpms/webkitgtk4/c/256d15f6647f1eb96adf30a688a640e135021ea7?branch=rawhide) and redirects to `webkit2gtk3`.

I'll update these references.

Reply:
I was curious where I got this from....it was apparently pre-existing in the tutorial before I climbed into this rabbit hole :)

https://github.com/beeware/beeware/blob/5165a027e606e607a57abea14427ac3bb7c65536/docs/tutorial/tutorial-0.rst?plain=1#L104

Reply:
ahh, i see....https://github.com/beeware/beeware/pull/50 used `webkitgtk4` since it was more widely available after the rename to `webkit2gtk3`. I think these concerns should be mitigated now given most of the unsupported distro versions are EOL.